cves/CVE-2015-6826.yml

```diff
@@ -355,8 +415,11 @@ CWE_instructions: |
 
   Just the number here is fine. No need for name or CWE prefix. If more than one
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
-CWE_note:
+CWE: 457
```

Comment:
Fantastic catch, the CWE noted on the NVD page seems to be a ways off.

Reply:
Glad you think so!